leetcode/src/d21/_2138_divide_a_string_into_groups_of_size_k.rs

```diff
@@ -0,0 +1,41 @@
+struct Solution;
+
+impl Solution {
+    pub fn divide_string(s: String, k: i32, fill: char) -> Vec<String> {
+        let k = k as usize;
+        let mut answer = s
```

Comment:
do it in two steps and remove the turbofish op.
let answer: Vec<Vec<char>> = ....collect();
let answer: Vec<Vec<String>> = ....collect();